nautobot_circuit_maintenance/views.py

```diff
@@ -204,7 +204,8 @@ def get_extra_context(self, request, instance):
             parsed_notification = models.ParsedNotification.objects.filter(raw_notification=instance).last()
         else:
             parsed_notification = None
-        return {"parsed_notification": parsed_notification}
+        raw_repr = instance.raw.tobytes().decode()
```

Comment:
Do we need exception handling for a potential case where `instance.raw` is *not* a binary-encoded string? Say if it's some arbitrary blob of binary data like (shudder) a PNG file?

Reply:
the data is gathered from email parts of types: "text/calendar" and "text/html". but adding an exception could save some future issue